core/templates/dev/head/domain/collection/CollectionUpdateService.js

```diff
@@ -99,6 +99,28 @@ oppia.factory('CollectionUpdateService', [
         return _getParameterFromChangeDict(changeDict, 'exploration_id');
       };
 
+      // Validates the tags for the collection
+      var _validateTags = function(tags) {
```

Comment:
We went with deciding against having underscores for private functions :)
Also, I don't think this helper function is necessary, since `isTagValid` does nothing except call this function, and `_validateTags` isn't used anywhere else. You can just put it in `isTagValid`.